projects/step-frontend/src/lib/modules/execution/components/alt-report-performance-overview-chart/alt-report-performance-overview-chart.component.html

```diff
@@ -14,12 +14,12 @@
     }
   </step-alt-report-widget-filter>
   <step-alt-report-widget-content>
-    @if (timeRange(); as range) {
+    @if (_state.timeRange$ | async; as range) {
       <step-standalone-dashlet
         [metricKey]="metricKey"
         [config]="chartConfig"
         [filters]="chartFilters()"
-        [timeRange]="range"
+        [timeRange]="range!"
```

Comment:
What is the reason to put `!` here?
If range will be null or undefined, @if statement won't show the component at all